README.md

```diff
@@ -535,3 +535,9 @@ In this particular example, `claim_to_map_user` is the claim that maps the OpenI
 
 To develop your own plugin, make sure to conform to the plugin interface defined in [interface.h](./plugins/user_mapping/include/irods/http_api/plugins/user_mapping/interface.h).
 Further documentation on the interface functions are within the file.
+
+## Secure Communication (SSL/TLS)
+
+Messages between a client and the HTTP API are always exchanged via an insecure communication channel. Securing the communication channel can be accomplished through the use of a proxy server.
```

Comment:
not a fan of the 'always ... insecure'...  how does this sound?

```suggestion
The HTTP API does not handle SSL/TLS termination itself.  Deployments should plan to provide a reverse proxy to handle SSL/TLS termination 'in front of' the HTTP API for secure communication with the client.
```

Reply:
Done.